packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -240,6 +255,7 @@ export async function createSDKConnection(params: CreateSDKConnectionParams) {
 
   if (IS_CLOUD) {
     await addCloudSDKMapping(connection);
+    await syncVercelSdkConnection(connection.organization);
```

Comment:
I think we should remove the `syncVercelSdkConnection` method from this file entirely.  We only want to do this when a resource is provisioned or upserted in the vercel controller, so it should be called there instead.  The code here fires for every SDK connection, not just the ones managed by vercel.